kafkatest/broker.go

```diff
@@ -161,7 +161,8 @@ type Producer struct {
 
 	// ResponseOffset is offset counter returned and incremented by every
 	// Produce method call. By default set to 1.
-	ResponseOffset int64
+	responseOffset int64
```

Comment:
Very minor thing: documentation should be updated to reflect the changed name :)